tools/test-proxy/Azure.Sdk.Tools.TestProxy/RecordingHandler.cs

```diff
@@ -464,7 +465,9 @@ public void SetDefaultExtensions(string recordingId = null)
             {
                 Sanitizers = new List<RecordedTestSanitizer>
                 {
-                    new RecordedTestSanitizer()
+                    new RecordedTestSanitizer(),
+                    new BodyKeySanitizer("$..access_token"),
```

Comment:
Can we add these as well - https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core.TestFramework/src/RecordedTestSanitizer.cs#L41-L45